starter-code/src/viking.js

```diff
@@ -1,8 +1,39 @@
 // Soldier
-function Soldier () {}
+function Soldier (health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+
+Soldier.prototype.attack = function(){
+  return (this.strength);
```

Comment:
Aquí estás pasando this.strength entre parentesis cómo si fuera un parametro por eso da error, simplemente quitando los parentesis debería funcionar sin problema.  
Sería algo así:   " return this.strength; " en lugar de: " return (this.strength); ".